HACKING.md

```diff
@@ -234,75 +234,7 @@ The chart is also available in the release artifact as a tarball.
-| DOCKER_REGISTRY           | Specify docker registry used in system tests                                         | docker.io                                        |
-| TEST_CLIENT_IMAGE         | Specify test client image used in systemtest                                         | docker.io/strimzi/test-client:latest-kafka-2.3.0 |
-| BRIDGE_IMAGE              | Specify kafka bridge image used in systemtest                                        | docker.io/strimzi/kafka-bridge:latest            |
-| TEST_LOG_DIR              | Directory for store logs collected during the tests                                  | ../systemtest/target/logs/                       |
-| ST_KAFKA_VERSION          | Kafka version used in images during the system tests                                 | 2.3.0                                            |
-| STRIMZI_DEFAULT_LOG_LEVEL | Log level for cluster operator                                                       | DEBUG                                            |
-| KUBERNETES_DOMAIN         | Cluster domain. It's used for specify URL endpoint of testing clients                | .nip.io                                          |
-
-If you want to use your own images with different tag or from different repository, you can use `DOCKER_REGISTRY`, `DOCKER_ORG` and `DOCKER_TAG` environment variables.
-
-`KUBERNETES_DOMAIN` should be specified only in case you are using specific configuration in your kubernetes cluster.
-
-##### Specific Kafka version
-
-To set custom Kafka version in system tests need to add system property `ST_KAFKA_VERSION` to one of the values in [kafka-versions](https://github.com/strimzi/strimzi-kafka-operator/blob/master/kafka-versions).
-
-##### Cluster Operator Log level
-
-To set the log level of Strimzi for system tests need to add system property `STRIMZI_DEFAULT_LOG_LEVEL` with one of the following values: `ERROR`, `WARNING`, `INFO`, `DEBUG`, `TRACE`.
-
-#### Test Cluster
-
-The integration and system tests are run against a cluster specified in the environment variable `TEST_CLUSTER_CONTEXT`. If this variable is not set, kubernetes client will use currently active context. Otherwise will use context from kubeconfig with name specified by `TEST_CLUSTER_CONTEXT` variable.
-
-For example command `TEST_CLUSTER_CONTEXT=remote-cluster ./systemtest/scripts/run_tests.sh` will execute tests with cluster context `remote-cluster`. However, since system tests use command line `Executor` for some actions, make sure that you are using context from `TEST_CLUSTER_CONTEXT`.
-
-System tests uses admin user for some actions. You can specify admin user via variable `TEST_CLUSTER_ADMIN` (by default it use `developer` because `system:admin` cannot be used over remote connections).
+System tests has its own guide with more information. See [Testing Guide](https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/TESTING.md) document for more information.
```

Comment:
Wouldn't it be better to have this in root and not under `systemtest`? I think it would be easier to find.

Reply:
Make sense

Reply:
I think you need to fix the link as well now ;-)

Reply:
fixed